pkgs/development/python-modules/mailrise/default.nix

```diff
@@ -0,0 +1,18 @@
+{ pkgs, lib, buildPythonPackage }:
+buildPythonPackage {
+  name = "mailrise";
+  pname = "mailrise";
+  version = "1.4.0";
+  propagatedBuildInputs = [ (pkgs.python3.withPackages (pythonPackages: with pythonPackages; [ apprise aiosmtpd pyaml ])) ];
+  src = pkgs.fetchgit {
+    url = "https://github.com/YoRyan/mailrise";
+    rev = "ee40be524f15a50653863983827baae4258d3e0f";
+    sha256 = "sha256-4y15hENo7tKLTamHKKJ94RES3XUldj1pe+VU6AnLKPE=";
+  };
+  meta = {
+    description = "An SMTP gateway for Apprise notifications";
```

Comment:
The description should not start with `An` or `The`
See: https://nixos.org/manual/nixpkgs/stable/#var-meta-description